crystal_toolkit/apps/examples/bs.py

```diff
@@ -0,0 +1,56 @@
+# standard Dash imports
+import dash
+import dash_html_components as html
+import dash_core_components as dcc
+
+# standard Crystal Toolkit import
+import crystal_toolkit.components as ctc
+
+# MPRester to get bs and dos
+from pymatgen import MPRester
+
+# create Dash app as normal
+app = dash.Dash()
+
+# If callbacks created dynamically they cannot be statically checked at app startup.
+# For this simple example this IS a problem and,
+# nested layout this will need to be enabled -- consult Dash documentation
+# for more information.
+# app.config["suppress_callback_exceptions"] = True
+
+# tell Crystal Toolkit about the app
+ctc.register_app(app)
+
+
+mpid = {"mpid": "mp-149"}
+
+with MPRester() as m:
+    bandstructure_symm_line = m.get_bandstructure_by_material_id(mpid["mpid"])
+    density_of_states = m.get_dos_by_material_id(mpid["mpid"])
+
+# create the Crystal Toolkit component
+structure_component = ctc.BandstructureAndDosComponent(
```

Comment:
e.g. put `ctcBandstructureAndDosComponent(..., id=)` here and reference that id in callbacks (also rename the variable so it's not `structure_component` anymore)

Reply:
Thanks! It is working now on my end.